### PR TITLE
Circleci

### PR DESCRIPTION
**Pull request by MiraAr**
PTAL

### CODE REVIEW

**Comment by MiraAr** [comment]
PTAL

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
       - node/with-cache:
           steps:
             - run: yarn install
-            - run: yarn run fix
```
Comment (louisrli) [inline .circleci/config.yml]: why did this get deleted

**Review by louisrli — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (MiraAr) [reply .circleci/config.yml:13]: we replaced fix by test because yarn fix did not work

Comment (MiraAr) [reply .circleci/config.yml]: because yarn fix did not work so we replace it by test

File: package.json [hunk 2]
```diff
@@ -24,10 +24,11 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "jest",
+    "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint": "eslint '**/*.{js,jsx}'",
-    "pretty": "prettier --write \"./**/*.{js,jsx}\""
+    "pretty": "prettier --write \"./**/*.{js,jsx}\"",
+    "fix": "!src/**/*/*.snap"
```
Comment (louisrli) [inline package.json]: this is not correct. you should combine yarn run pretty and yarn run fix into one command. please work with Abdul rahman or ammar or ask halit

File: .circleci/config.yml (hunk 1 above)
Comment (louisrli) [reply .circleci/config.yml]: this file should have no changes (revert the changes) it is correct as it is

**Comment by MiraAr** [comment]
PTAL

**Comment by louisrli** [comment]
LGTM @OmarAlithawi review then merge

**Review by louisrli — APPROVED** [review]

**Review by OmarAlithawi — APPROVED** [review]

**Pull request merged**